src/server/ApiDispatchers.cpp

```diff
@@ -798,6 +912,13 @@ static DWORD TraceGetThreadId(CONSOLE_API_MSG* const m)
 
     size_t written;
     std::span<const INPUT_RECORD> buffer(reinterpret_cast<INPUT_RECORD*>(pvBuffer), cbSize / sizeof(INPUT_RECORD));
+
+    TraceConsoleAPICallWithOrigin(
+        "WriteConsoleInput",
+        TraceLoggingBoolean(a->Unicode, "Unicode"),
+        TraceLoggingBoolean(a->Append, "Append"),
+        TraceLoggingUIntPtr(buffer.size(), "Buffer"));

```

Comment:
we may want to call these size fields `Records` or something, right? since they're not the actual buffer